app/create.py

```diff
@@ -385,11 +386,25 @@ def get(self):
             add_profile_icon_url(website, self.transitionary_get_url)
             for website in self.config.profile_websites or []]
         self.render('create.html',
+                    captcha_html=self.get_captcha_html(),
                     profile_websites=profile_websites,
                     profile_websites_json=simplejson.dumps(profile_websites),
                     onload_function='view_page_loaded')
 
     def post(self):
+        captcha_response = self.get_captcha_response()
+        if not captcha_response.is_valid:
+            self.params.create_mode = True
```

Comment:
Can you factor out line 397-405 as a helper method as it is duplicate of get() method?

Reply:
Done.  _render_create_html.